doc/classes/CapsuleMesh.xml

```diff
@@ -11,12 +11,14 @@
 	<members>
 		<member name="height" type="float" setter="set_height" getter="get_height" default="2.0">
 			Total height of the capsule mesh (including the hemispherical ends).
+			[b]Note:[/b] The [member height] of a [CapsuleMesh] must be at least twice the [member radius]; otherwise, the capsule becomes a circle. If the [member height] is less than twice the [member radius], the properties adjust to a valid value.
```

Comment:
It _should_ be possible to summarize this note much more, but I can't think of anything as of writing this.
Using "capsule" makes the note more broad, referring to all capsule-like shapes in the engine.
```suggestion
			[b]Note:[/b] The [member height] of a capsule must be at least twice its [member radius]. Otherwise, the capsule becomes a circle. If the [member height] is less than twice the [member radius], the properties adjust to a valid value.
```
If you change this, this should be applied to all notes like this in this PR.